gramex/handlers/mlhandler.py

```diff
@@ -369,9 +452,29 @@ def post(self, *path_args, **path_kwargs):
         action = self.args.pop('_action', 'predict')
         if action not in ACTIONS:
             raise HTTPError(BAD_REQUEST, f'Action {action} not supported.')
-        res = yield gramex.service.threadpool.submit(getattr(self, f"_{action}"))
-        self.write(json.dumps(res, indent=2, cls=CustomJSONEncoder))
+        if self.backend == "transformers":
+            data = self._parse_data(_cache=False)
+            move_to_cpu(self.model)
+            kwargs = {}
+            if action == 'train':
+                kwargs = self._coerce_transformers_opts()
+                kwargs['model_path'] = self.model_path
+                args = _train_transformer, self.model, data
+            elif action == 'score':
+                args = _score_transformer, self.model, data
+            elif action == 'predict':
+                args = self.model, data['text'].tolist()
+            res = yield gramex.service.threadpool.submit(*args, **kwargs)
+            self.write(json.dumps(res, indent=2, cls=CustomJSONEncoder))
+        else:
+            res = yield gramex.service.threadpool.submit(getattr(self, f"_{action}"))
+            self.write(json.dumps(res, indent=2, cls=CustomJSONEncoder))      
         super(MLHandler, self).post(*path_args, **path_kwargs)
+            
+    def get_cached_arg(self, argname):
```

Comment:
It turns out that we don't need this function either. It's not being used anywhere. Please remove this as well.